core/gdxsv/gdxsv_emu_hooks.cpp

```diff
@@ -9,11 +9,97 @@
 #include "oslib/oslib.h"
 #include "rend/gui_util.h"
 #include "version.h"
+#include <fstream>
+#include "log/LogManager.h"
+
+#ifdef _WIN32
+#define CHAR_PATH_SEPARATOR '\\'
+#else
+#define CHAR_PATH_SEPARATOR '/'
+#endif
 
 void gdxsv_latest_version_check();
 static bool gdxsv_update_available = false;
 static std::string gdxsv_latest_version_tag;
 
+void gdxsv_flycast_init() {
+	std::ifstream fs;
+	fs.open(get_writable_data_path("crash_dmp_list.txt"));
+	std::vector<std::string> unhandled_dmp = {};
+	if (fs.is_open()) {
+		std::string line;
+		
+		while (std::getline(fs, line)) {
+			std::size_t found = line.find_last_of(",");
+			if (found == std::string::npos) continue;
+			std::string file_path = line.substr(0, found);
+			std::string git_version = line.substr(found+1);
+			
+			std::vector<UploadField> fields = {};
+			
+			fields.push_back({"upload_file_minidump", file_path, "application/octet-stream"});
+			
+			std::string log = file_path;
+			if (log.find(".dmp") != std::string::npos) {
+				log.replace(log.find(".dmp"), sizeof(".dmp") - 1, ".log");
+				fields.push_back({"flycast_log", log, "text/plain"});
+			}
+			fields.push_back({"emu_cfg", get_writable_config_path("emu.cfg"), "text/plain"});
+			fields.push_back({"sentry[release]", "", "", git_version});
+			
+			std::string minidump_upload_url;
+			if (git_version.find("dev") == std::string::npos)
+				minidump_upload_url = "https://o4503934635540480.ingest.sentry.io/api/4503960868683776/minidump/?sentry_key=6fd422fe4ade467c842416de430a9968";
+			else
+				minidump_upload_url = "https://o4503934635540480.ingest.sentry.io/api/4503960859443200/minidump/?sentry_key=1bcd9bcca32a46c888244b392b4dc6eb";
+			
+			int result = os_UploadFilesToURL(minidump_upload_url, fields);
+			NOTICE_LOG(COMMON, "Upload status: %d, %s", result, file_path.c_str());
+			if (result != 200) {
+				unhandled_dmp.push_back(line);
```

Comment:
Since crash logs are stored to the Temp directory, the file may have disappeared after restarting.
In that case, the transmission does fail every time?